build.gradle

```diff
@@ -35,6 +35,13 @@ import org.gradle.util.DistributionLocator
 import java.nio.file.Files
 import java.nio.file.Path
 import java.security.MessageDigest
+plugins {
+    id 'com.gradle.build-scan' version '1.13.2'
+}
+buildScan {
```

Comment:
I think we should wrap this in a property check so that the default is to not add this. I don't think we should access the terms of service for anyone (ie non elastic employees) running our build. We (developers) can then set this property in our `~/.gradle/gradle.properties`.